code/modules/uplink/uplink_items.dm

```diff
@@ -80,15 +81,18 @@
 	/// Weight of the category. Used to determine the positioning in the uplink. High weight = appears first
 	var/weight = 0
 
+/// Returns by how much percentage do we reduce the price of the selected item
 /datum/uplink_item/proc/get_discount()
-	return pick(4;0.75,2;0.5,1;0.25)
+	return pick(4;0.25,2;0.5,1;0.75)
```

Comment:
I tried it out and sadly, turns out numbers can't be keys in associative lists in byond. It could work if I stored them as strings, and converted those to numbers, which would be a rather ugly...

Reply:
If i still have to do this, would the keys being string defines, which get evaulated in a switch statement work? Much better than having the number as a string that gets turned back into a number, and the defines would make it clear that a bigger number is a bigger discount after my change.